### DISCUSSION TITLE
Can I disable the view of plain text and html?

### DISCUSSION THREAD

**Discussion by daniagongora** [postalserver/postal#3154] [Help with using Postal]
I have configured the message retention in Postal as follows: I have set 0 days for the storage of raw message data and 1 day for the availability of message metadata. However, I have noticed that I can still see the plain text and HTML content of older messages.
For privacy of the message, I would like to know if there is any additional configuration that can restrict the visibility of this data, or if there is a reason why my current settings are not working.

My Postal is recently updated to 3.3.4

**Comment by willpower232** [comment]
3.3.4 fixed the message pruning task so if recently was yesterday then maybe today or tomorrow the amount of data will sort itself out. I'm not sure how well 0 days of raw message data will work since it needs to be stored briefly.

This will also prevent you from knowing about and dealing with held messages, unless you've set up webhooks to know what is happening.

Arguably the best option is to not give logins to people you don't trust :sweat_smile: presuming that is what you are trying to do.

**Reply by daniagongora** [reply]
Ok, thank you so much for the anwer!
